### PR TITLE
Created Installer and Supporting Docs

### PR DESCRIPTION
**Pull request by noodlemctwoodle**
Created easy installer without the need to run Powershell and reduce
peoples security by disabling script execution policy. Installer.exe
is a simple light weight installer the puts all the correct files in
the correct location. 

The following changes have been added to the Repo.

- Added Installer.exe
- Updated Readme.md with instructions for installing with Installer.exe
- Added Wiki entry for compiling installer yourself (trust but verify)
- Added compiler script for open source compiler Inno Setup 5.6.1

Resubmitting Pull Request with squashed changes and cleaner commits.

### CODE REVIEW

**Comment by mikefitz888** [comment]
Install.exe :: Scan Results
[VirusTotal (0 / 69)](https://www.virustotal.com/#/url/4986e0ba54e1b5404469bb80ea6e3a891e36b7216d5bfdb22a2e53cc003e4ebc/detection) 👍

**Comment by mikefitz888** [comment]
I cannot see the wiki entry yet

**Comment by noodlemctwoodle** [comment]
You may need to carry it across from my [fork](https://github.com/noodlemctwoodle/Azure-Manic-7Pace/wiki/Compile-Inno-Installer), I can give you the code.

File: inno_compile-script [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+; (To generate a new GUID, click Tools | Generate GUID inside the IDE.)
+AppId={{447729A4-3AA8-4B49-B6DB-047D48668268}
+AppName={#MyAppName}
+AppVersion={#MyAppVersion}
+;AppVerName={#MyAppName} {#MyAppVersion}
+AppPublisher={#MyAppPublisher}
+AppPublisherURL={#MyAppURL}
+AppSupportURL={#MyAppURL}
+AppUpdatesURL={#MyAppURL}
+DefaultDirName={localappdata}\Finkit\ManicTime\Plugins\MFitzpatrick.TagSource.Azure7PacePlugin
```
Comment (mikefitz888) [inline inno_compile-script]: This should be `...\Plugins\Packages\MFitz...`

File: readme.md [hunk 2]
```diff
@@ -15,19 +15,13 @@ It achieves this in 2 ways:
...
-to `%systemdrive%\Users\%username%\AppData\Local\Finkit\ManicTime\Plugins`.
-
-First step is to locate the ManicTime plugin directory. To do so, open ManicTime. Click on the cog at the top right of the window, and click
-`Advanced > Open Datebase Folder` in the menu that appears. You will see a folder here called `Plugins`, navigate to `Plugins\Packages`. Here is where you will copy the plugin folder. If `\Packages` does not exist, create it.
-
-The plugin folder is located within the `\dist` folder of this repository.
-Close ManicTime (The application can be hidden in the [Notification Area](https://docs.microsoft.com/en-us/windows/desktop/uxguide/winenv-notification), so ensure it has been completely exited)
-Copy the folder named `MFitzpatrick.TagSource.Azure7PacePlugin` to the plugin folder opened in the previous step.
+Download Install.exe and run as administrator. This will extact the plugin to the correct plugins directory for Manic Time.
+ `{localappdata}\Finkit\ManicTime\Plugins\MFitzpatrick.TagSource.Azure7PacePlugin`
```
Comment (mikefitz888) [inline readme.md]: This should be updated inline with the correct install location.

File: Installer/Inno_Setup_Compiler_Script.iss [hunk 3]
```diff
@@ -15,12 +12,12 @@ AppPublisher={#MyAppPublisher}
 AppPublisherURL={#MyAppURL}
 AppSupportURL={#MyAppURL}
 AppUpdatesURL={#MyAppURL}
-DefaultDirName={localappdata}\Finkit\ManicTime\Plugins\MFitzpatrick.TagSource.Azure7PacePlugin
+DefaultDirName={localappdata}\Finkit\ManicTime\Plugins\Packages\MFitzpatrick.TagSource.Azure7PacePlugin
 DefaultGroupName={#MyAppName}
 AllowNoIcons=yes
-OutputDir=C:\Users\{USERNAME}\Downloads\Azure-Manic-7Pace-master
+OutputDir=C:\Users\Toby.Goulden\Downloads\Azure-Manic-7Pace-master
```
Comment (mikefitz888) [inline Installer/Inno_Setup_Compiler_Script.iss]: You can't just assume everyone has the same name 😛

File: Installer/Inno_Setup_Compiler_Script.iss [hunk 4]
```diff
@@ -15,12 +12,12 @@ AppPublisher={#MyAppPublisher}
...
 OutputBaseFilename=Install
-SetupIconFile=C:\Windows\ACU.ico
+SetupIconFile=C:\Users\{USERNAME}\Downloads\Azure-Manic-7Pace-master\icon.ico
```
Comment (mikefitz888) [inline Installer/Inno_Setup_Compiler_Script.iss:20]: Shouldn't this be a relative path?

File: Installer/Inno_Setup_Compiler_Script.iss [hunk 5]
```diff
@@ -30,4 +27,4 @@ Name: "english"; MessagesFile: "compiler:Default.isl"
 [Files]
 Source: "C:\Users\{USERNAME}\Downloads\Azure-Manic-7Pace-master\MFitzpatrick.TagSource.Azure7PacePlugin\PluginIcon.png"; DestDir: "{app}"; Flags: ignoreversion
 Source: "C:\Users\{USERNAME}\Downloads\Azure-Manic-7Pace-master\MFitzpatrick.TagSource.Azure7PacePlugin\PluginSpec.json"; DestDir: "{app}"; Flags: ignoreversion
-Source: "C:\Users\{USERNAME}\Downloads\Azure-Manic-7Pace-master\MFitzpatrick.TagSource.Azure7PacePlugin\Lib\*"; DestDir: "{localappdata}\Finkit\ManicTime\Plugins\MFitzpatrick.TagSource.Azure7PacePlugin\lib"; Flags: ignoreversion
+Source: "C:\Users\{USERNAME}\Downloads\Azure-Manic-7Pace-master\MFitzpatrick.TagSource.Azure7PacePlugin\Lib\*"; DestDir: "{localappdata}\Finkit\ManicTime\Plugins\Packages\\MFitzpatrick.TagSource.Azure7PacePlugin\lib"; Flags: ignoreversion
```
Comment (mikefitz888) [inline Installer/Inno_Setup_Compiler_Script.iss]: Double backslash `\\`

File: Installer/Inno_Setup_Compiler_Script.iss (hunk 4 above)
Comment (mikefitz888) [reply Installer/Inno_Setup_Compiler_Script.iss:20]: Nevermind, I think i mis-understood how it works.

**Comment by mikefitz888** [comment]
Install.exe :: Scan Results (For update 2)
[VirusTotal (0/69)](https://www.virustotal.com/#/url/b27ea20b52b4d53e9267e7cba08ea4f136687c4aff05d5da9643e8eeb07a0a94/detection)

File: Installer/Inno_Setup_Compiler_Script.iss (hunk 5 above)
Comment (noodlemctwoodle) [reply Installer/Inno_Setup_Compiler_Script.iss]: Removed double backslash, updated PR

File: Installer/Inno_Setup_Compiler_Script.iss (hunk 3 above)
Comment (noodlemctwoodle) [reply Installer/Inno_Setup_Compiler_Script.iss]: Removed username from the file path.

File: inno_compile-script (hunk 1 above)
Comment (noodlemctwoodle) [reply inno_compile-script]: Updated to the correct path added in updated PR

File: readme.md (hunk 2 above)
Comment (noodlemctwoodle) [reply readme.md]: Now has the correct path in the latest PR.

**Pull request merged**